tests/components/sensor/test_rmvtransport.py

```diff
@@ -97,77 +100,90 @@ def get_departuresMock(stationId, maxJourneys,
     return data
 
 
-def get_errDeparturesMock(stationId, maxJourneys,
-                          products):  # pylint: disable=invalid-name
+def get_no_departures_mock():
+    """Mock rmvtransport departures loading."""
```

Comment:
Stale docstring.

Reply:
Well spotted and thank you so much for reviewing.